android/app/src/main/assets/config.json

```diff
@@ -1,5 +1,5 @@
 {
   "Email": "dev@fossasia.org",
   "App_Name": "Open Event",
-  "Api_Link": "https://eventyay.com/api/v1/events/6/"
+  "Api_Link": "http://open-event-dev.herokuapp.com/api/v1/events/6/"
```

Comment:
I changed it to test it on development deployment. Actually i forgot to change it back. Will change it :+1: